bin/run-command.sh

```diff
@@ -18,10 +18,101 @@ set -euo pipefail
 # TODO: Add ability to change task IAM Role. Part 3 of multipart update https://github.com/navapbc/template-infra/issues/354#issuecomment-1693973424
 # TODO: Change to keyword arguments. Part 3 of multipart update https://github.com/navapbc/template-infra/issues/354#issuecomment-1693973424
 
-APP_NAME="$1"
-ENVIRONMENT="$2"
-COMMAND="$3"
-ENVIRONMENT_VARIABLES=${4:-""}
+ACCOUNT_ID="$(./bin/current-account-id.sh)"
+
+while :; do
+  case $1 in
+    -a|--app|--app-name)
+      if [ "$2" ]; then
+        APP_NAME=$2
+        shift
+      else
+        die "$1 flag present without app name"
+      fi
+      ;;
+    -e|--env|--env-name|--environment)
```

Comment:
app name, environment, and command aren't optional, could we keep them as positional parameters?

Reply:
It would add more complication to handle both positional and flag based arguments but I can give it a shot

Reply:
If it's more complicated then we can keep it the way it is. I didn't know if there was a simple way to handle both

Reply:
what about something like

```
while :; do
  case $1 in
    --role-name)
      ...
    --environment-variables)
      ...
    *)
      APP_NAME=$1
      ENVIRONMENT=$2
      COMMAND=$3
      break
      ;;
done
```

Reply:
Just tested it out, this seems to work:

```bash
while :; do
  case $1 in
    --environment-variables)
      ENVIRONMENT_VARIABLES=$2
      shift 2
      ;;
    --role-name)
      ROLE_NAME="$2"
      shift 2
      ;;
    *)
      APP_NAME="$1"
      ENVIRONMENT="$2"
      COMMAND="$3"
      break
      ;;
  esac
done

echo "APP_NAME=$APP_NAME"
echo "ENVIRONMENT=$ENVIRONMENT"
echo "COMMAND=$COMMAND"
echo "ENVIRONMENT_VARIABLES=${ENVIRONMENT_VARIABLES:-}"
echo "ROLE_NAME=${ROLE_NAME:-}"
```

<img width="942" alt="image" src="https://github.com/navapbc/platform-test/assets/447859/680cae02-d345-43c7-b4e4-fe5f2fab7aaf">

Reply:
I'll give that a shot after I figure out some of the other issues